post_processing/ci_plots.jl

```diff
@@ -590,22 +623,112 @@ EDMFBoxPlots = Union{
     Val{:prognostic_edmfx_trmm_column},
 }
 
+"""
+    plot_edmf_vert_profile!(grid_loc, var_group)
+
+Helper function for `make_plots_generic`. Takes a list of variables and plots
+them on the same axis.
+"""
+function plot_edmf_vert_profile!(grid_loc, var_group)
+    z = var_group[1].dims["z"]
+    units = var_group[1].attributes["units"]
+    ax = CairoMakie.Axis(
+        grid_loc[1, 1],
+        ylabel = "z [$(var_group[1].dim_attributes["z"]["units"])]",
+        xlabel = "$(short_name(var_group[1])) [$units]",
+        title = var_info(var_group[1]),
+    )
+
+    for var in var_group
+        CairoMakie.lines!(ax, var.data, z, label = short_name(var))
+    end
+    length(var_group) > 1 && Makie.axislegend(ax)
+end
+
+
+"""
+    plot_custom_title!(grid_loc, var)
+
+Helper function for `make_plots_generic`. Plots a variable with a custom title.
+"""
+plot_custom_title!(grid_loc, var) = viz.plot!(
+    grid_loc,
+    var;
+    more_kwargs = Dict(:axis => ca_kwargs(title = var_info(var))),
+)
+
+"""
+    group_edmf_vars(vars)
+
+Groups updraft and gridmean EDMF OutputVars into a Dict.
+Groupings are matched via variable short name, location, and time (if applicable).
+"""
+function group_edmf_vars(vars)
+    grouped_vars = OrderedDict{String, Vector{ClimaAnalysis.OutputVar}}()
+    for var in sort(vars, by = var_info)
+        suffix = var_info(var)
+        if haskey(grouped_vars, suffix)
+            push!(grouped_vars[suffix], var)
+        else
+            grouped_vars[suffix] = [var]
+        end
+    end
+    return grouped_vars
+end
+
 function make_plots(::EDMFBoxPlots, simulation_path)
     simdir = SimDir(simulation_path)
 
-    short_names = ["ua", "wa", "thetaa", "taup", "haup", "waup", "tke", "arup"]
+    short_names = [
+        "ua",
+        "wa",
+        "thetaa",
+        "thetaaup",
+        "ta",
+        "taup",
+        "ha",
+        "haup",
+        "waup",
+        "tke",
+        "arup",
+        "hus",
+        "husup",
+        "hur",
+        "hurup",
+        "cl",
+        "clw",
+        "clwup",
+        "cli",
+        "cliup",
+    ]
     reduction = "inst"
     period = "30m"
+
     vars = [
         get(simdir; short_name, reduction, period) for short_name in short_names
     ]
```

Comment:
Pseudo code
```suggestion
    vars = [
        get(simdir; short_name, reduction, period) for short_name in short_names
    ]
    grouped_vars = Any[]
    for index, short_name in enumerate(short_names)
        # skip updraft variables
        short_name[end-2:end] == "up" && continue

        up_index = find(short_name * "up" in short_names)
        if isnothing(up_index)
           add_tup = tuple(vars[index])
        else
           add_tup = tuple(vars[index], vars[up_index])
         end  
            push!(grouped_vars, add_tup)
    end
```
`grouped_vars` is a vector of 1/2-tuples of `OutputVars` that you can slice.